prime-router/src/main/kotlin/Report.kt

```diff
@@ -76,7 +76,8 @@ data class ReportStreamFilterResult(
     val filterArgs: List<String>,
     val filteredTrackingElement: String,
     val filteredIndex: Int,
-    override val type: ActionLogDetailType = ActionLogDetailType.TRANSLATION
+    override val type: ActionLogDetailType = ActionLogDetailType.TRANSLATION,
+    val filterType: String? = ""
```

Comment:
Flip these two arguments allowing the `override` to remain the last argument, this makes calling the function easier since it will usually have a filterType.